manifests/templates/decco-deployment.yaml.tmpl

```diff
@@ -1,9 +1,12 @@
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: decco-operator
 spec:
   replicas: 1
+  selector:
+    matchLabels:
+      name: decco-operator
```

Comment:
Since you are at it, maybe rename the key of this label to `k8s-app` for consistency and unusualness of `name` as a key.

Reply:
I buy that; added.